.storybook/components/Container/index.js

```diff
@@ -3,9 +3,12 @@
  * @copyright IBM Security 2019
  */
 
-// TODO: Modular styles - Revert.
+// TODO: `modular-styles` - Revert.
```

Comment:
What does this syntax do? Thanks

Reply:
It's to keep it in line with the other comments for this branch so we can do a 'find all' later to address the outstanding TODOs